src/sparsezoo/model/utils.py

```diff
@@ -73,11 +119,20 @@ def load_files_from_directory(directory_path: str) -> List[Dict[str, Any]]:
     return files
 
 
+def _get_compressed_size(request_json: List[Dict[str, Any]]) -> int:
+    compressed_file_name = "model.onnx.tar.gz"
+    for file in request_json:
```

Comment:
nit: list comprehension

Reply:
IMO the recommended idiom would not make the code cleaner so leaving this as is